source/nvwave.py

```diff
@@ -1070,3 +1070,7 @@ def initialize():
 		func.restype = HRESULT
 		func.errcheck = _wasPlay_errcheck
 	NVDAHelper.localLib.wasPlay_startup()
+
+
+def usingWasapiWavePlayer() -> bool:
+	return issubclass(WavePlayer, WasapiWavePlayer)

```

Comment:
shouldn't this be

```suggestion
	return isinstance(WavePlayer, WasapiWavePlayer)
```

issubclass checks the inheritance of a class, not an instance

Reply:
No. `WavePlayer` and `WasapiWavePlayer` are both classes.

In `nvwave.py`, `WavePlayer` is set to `WinmmWavePlayer` and then in `nvwave.initialize`, `WavePlayer` is set to `WasapiWavePlayer` if WASAPI is enabled.

I could have written `WavePlayer == WasapiWavePlayer` instead, but I felt using `issubclass` was more robust in case one makes a subclass of `WasapiWavePlayer` in the future for whatever reason.

Reply:
my mistake, thanks